src/main.ts

```diff
@@ -33,6 +33,19 @@ async function main() {
     const currentAppProperty = await client.containerApps.get(taskParams.resourceGroup, taskParams.containerAppName);
 
     // TBD: Remove key when there is key without value
+
+    let traffics = [];
+    currentAppProperty.configuration!.ingress!.traffic!.forEach((traffic: TrafficWeight) => {
+      if (traffic.weight > 0) {
```

Comment:
I can see this warning on VScode.
![image](https://user-images.githubusercontent.com/4566555/190536406-a0e5e9a0-76e0-4df7-b35c-f9c7197e8c52.png)

The following condition is more reliable.
```js
      if (traffic.weight && traffic.weight > 0) {
```